src/main/webapp/app/entities/participation-submission/participation-submission.component.html

```diff
@@ -0,0 +1,51 @@
+<h2 *ngIf="participation">Submissions of {{ participation.student.firstName }} for Exercise {{ participation.exercise.title }}</h2>
```

Comment:
```suggestion
<h2 *ngIf="participation">Submissions of {{ participation.student.name }} for Exercise {{ participation.exercise.title }}</h2>
```

Reply:
To use this, you have to add the following attribute into account.model.ts

`public name: string | null;`

Reply:
I just added this in `develop`, will be available after updating this branch from develop